### PR TITLE
OpenVINO IR module

### PR DESCRIPTION
**Pull request by cih9088**
### Related MPA [PR](https://github.com/openvinotoolkit/model_preparation_algorithm/pull/95)
- add requirements for omz_downloader/omz_converter

### CODE REVIEW

**Comment by harimkang** [comment]
Hi, in the PR of the `feature/otx` target, there is no need to update `external/model-preparation-algorithm/submodule`.
The environment of `feature/otx` uses mpa's otx branch to install **directly**.

**Comment by goodsong81** [comment]
> Hi, in the PR of the `feature/otx` target, there is no need to update `external/model-preparation-algorithm/submodule`. The environment of `feature/otx` uses mpa's otx branch to install **directly**.

But, it is convenient for development :)
Let's do it until we could finally remove the dependency.

**Comment by harimkang** [comment]
> 

Yes then I don't have any special comments on this PR. If the PR on the mpa side is merged, I think we can update and merge!

**Comment by harimkang** [comment]
@cih9088 PR on the MPA side has been merged, and submodules in otx branch have been removed from the recent work. Please reflect. :)

**Comment by cih9088** [comment]
@harimkang done.

**Review by harimkang — DISMISSED** [review]
LGTM! I think we can check the test based on this PR and merge if there are no problems.

File: requirements/openvino.txt [hunk 1]
```diff
@@ -4,4 +4,4 @@ nncf==2.3.0
 onnx==1.10.1
 openmodelzoo-modelapi==2022.2.0
 openvino==2022.2.0
-openvino-dev==2022.2.0
+openvino-dev[tensorflow2,mxnet,caffe]==2022.2.0
```
Comment (goodsong81) [inline requirements/openvino.txt:7]: Just curiosity, above 3 all needed for download feature?

**Review by goodsong81 — DISMISSED** [review]
LGTM. Thank you for the great feature.

File: requirements/openvino.txt (hunk 1 above)
Comment (cih9088) [reply requirements/openvino.txt:7]: Yes, indeed. Please refer the **Implementation** column of the [list](https://github.com/openvinotoolkit/open_model_zoo/blob/master/models/public/index.md#classification-models). Note that **paddle**  library would be required as well if we wanted to support it.

**Comment by harimkang** [comment]
@cih9088 There was a problem in the CI test.. but it was resolved by merging the corresponding [PR](https://github.com/openvinotoolkit/training_extensions/pull/1444). Could you please rebase the latest feature/otx commit for CI test checking?

**Comment by cih9088** [comment]
@harimkang done.

**Review by goodsong81 — APPROVED** [review]

**Review by harimkang — APPROVED** [review]

**Pull request merged**